### ISSUE TITLE
Error message about the end of life of org.freedesktop.Platform 22.08 (for Minder)

### ISSUE THREAD

**Issue by qua-rus** [phase1geo/Minder#633]
```sh
Info: runtime org.freedesktop.Platform branch 22.08 is end-of-life, with reason:
   org.freedesktop.Platform 22.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this runtime:
   com.github.phase1geo.minder
```

**Comment by lxwulf** [comment]
I have a similar message but from the `GNOME Software` application.

The message appears only on the `flatpak` package and only from the source of `flathub`. So I guess the source of the message and the message itself is the same. 

I have a `fedora linux 40` and it has its own `flatpak` source named `registry.fedoraproject.org`. The `RPM` package is also without any warning messages.

![image](https://github.com/user-attachments/assets/9801fb43-a099-4e4b-9466-8a511dbf02c2)

**Comment by CodeShakingSheep** [comment]
Same here for the flatpak.
![image](https://github.com/user-attachments/assets/6d43b64a-1ab1-460e-90bb-0053cf46f67f)

**Comment by phase1geo** [comment]
I will have this issue resolved in the 1.17 release of Minder.

**Comment by CodeShakingSheep** [comment]
Just to let you know @phase1geo , this issue has already been resolved for me with the closing of https://github.com/flathub/com.github.phase1geo.minder/issues/39 (specifically merging of https://github.com/flathub/com.github.phase1geo.minder/pull/40/files). 

Warning is gone and Freedesktop Platform 22.08 was automatically removed on my system after the update.
![image](https://github.com/user-attachments/assets/5b8f7162-2cdf-499f-a3cc-13ccfb999881)

Thank you!
